### ISSUE TITLE
Difference in border-color between selected ChoiceGroup item and ChoiceGroup item with keyboard focus is not very distinguishable

### ISSUE THREAD

**Issue by shyama1208** [OfficeDev/office-ui-fabric-react#2459]
This happens in non-high contrast mode.
When a choiceGroup item is selected and we navigate to that item using keyboard then the difference in border-color is very less.

**Comment by kristoferbrown** [comment]
@shyama1208, I'm not sure I'm clear on what the issue is here. Focused choiceGroup items have a black border and selected ones do not.

Can you provide screenshots, gifs, or more explanation to clarify what the issue is?

**Comment by shyama1208** [comment]
When choiceGroup item is selected it has blue border
![image](https://user-images.githubusercontent.com/26054709/29651465-86b4dfc8-88be-11e7-80f8-1d21213021fc.png)


When selected choicegroup item is in focus:
![image](https://user-images.githubusercontent.com/26054709/29651498-ab11ef78-88be-11e7-9fbf-49672a54a9dc.png)


The difference in the border color for both the images is not very significant.

**Comment by kristoferbrown** [comment]
@betrue-final-final, is the similarity between these two borders problematic enough to need fixing? If so, what should the borders look like instead?

**Comment by betrue-final-final** [comment]
There should be a gap between the focus rectangle and the selection rectangle. @lynamemi might be working on a fix, or be able to put it into her pending PR.

![image](https://user-images.githubusercontent.com/20363037/35399239-a8fb32d6-01a8-11e8-8eed-c7f0f7c271e0.png)

**Comment by lynamemi** [comment]
I'll group this in with a high contrast mode focus fix: https://github.com/OfficeDev/office-ui-fabric-react/issues/3621

**Comment by lynamemi** [comment]
Since this is being tracked here: https://github.com/OfficeDev/office-ui-fabric-react/issues/3621, I'll close this issue.

Here is the PR that will fix this: https://github.com/OfficeDev/office-ui-fabric-react/pull/4572
